proto/storage/cve.proto

```diff
@@ -29,6 +29,13 @@ enum Source {
   SOURCE_NVD = 3;
 }
 
+//Added this score version for nvd cvss score. moving out and re using score version from cveinfo will cause backward compatibility issue
+enum CvssScoreVersion {
+  V2 = 0; // No unset for automatic backwards compatibility
+  V3 = 1;
+  UNKNOWN_VERSION = 2;
```

Comment:
Keep UNKOWN_VERSION as index 0 in the enum followed by V2 and V3. That will avoid having to explicitly set version to UNKOWN_VERSION in tests and other places. Also, it will keep the list ordered when other score versions are added later.